.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!---
+Thank you for submitting a pull request to AppFlowy. The team will dedicate their best efforts to reviewing and approving your pull request. If you have any questions about the project or feedback for us, please join our [Discord](https://discord.gg/wdjWUXXhtw).
+-->
+
+<!---
+If your pull request adds a new feature, please drag and drop a video into this section to showcase what you've done! If not, you may delete this section.
+-->
+
+### Feature Preview
+
+<!---
+List at least one issue here that this PR addresses. If it fixes the issue, please use the [fixes](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) keyword to close the issue. For example:
+fixes https://github.com/AppFlowy-IO/AppFlowy/pull/2106
```

Comment:
You can also just use Git keywords like `Relates` `Closes`, etc. and use the github id `#1234`